BackportInstructions.md

```diff
@@ -155,6 +155,8 @@ For reference, here are a couple of assemblies that have been addressed:
 
     i. The tool should be able to preserve double slash comments if they were placed on top of a public API, and will add the new triple slash comments separately. Make sure to verify that no important information is lost, especially the kind that is directed to developers/maintainers.
 
+    j. Docs repos indicate generic type parameters using `` `1 ``, `` `2 ``, etc., but in triple slash, they are represented with `{T}`, `{T, U}`, etc. The tool currently just converts them all to `{T}`, but it may not be the correct type parameter name or number of parameters, so make sure it's correct.
```

Comment:
Did you test this with more than one type parameter?  It might create an error.  You might be better off synthesizing generic parameter names (T, U, V, ...) unless your goal is to force people to look at the error.